packages/web-components/fast-foundation/src/slider-label/slider-label.ts

```diff
@@ -17,6 +17,9 @@ const defaultConfig: SliderConfiguration = {
     disabled: false,
 };
 
+const heightNumber =
```

Comment:
Why is this in fast-foundation?

Reply:
doh! left behind after the move of styles, will fix

Reply:
removed